nvflare/app_common/pt/pt_multi_process_executor.py

```diff
@@ -26,5 +27,14 @@ def get_multi_process_command(self) -> str:
             f"{sys.executable} -m torch.distributed.run --nproc_per_node="
             + str(self.num_of_processes)
             + " --nnodes=1 --node_rank=0"
-            + ' --master_addr="localhost" --master_port=1234'
+            + ' --master_addr="localhost" --master_port='
+            + str(self._get_open_port())
         )
+
+    def _get_open_port(self):
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.bind(("", 0))
+        s.listen(1)
```

Comment:
Not sure if listen is necessary.  The kernel seems able to find a free port after trying to bind to port 0.  Other than that, it's perfect.

Reply:
bind() will find the open port. listen() is to set the connection queue. May not be absolutely necessary, but a good practice.